File: Linguagem C++/Vetores/MaiorPosicao.cpp
#include <bits/stdc++.h>

using namespace std;

int main()
{
    int n,posMaior;
    double maior;

    cout << "Quantos numeros voce vai digitar? ";
    cin >> n;

    double vet[n];

    //Lendo o vetor
    for (int i=0;i<=n-1;i++){
        cout << "Digite um numero: ";
        cin >> vet[i];
    };

    cout << endl;
    //Achando o maior valor e sua posição
    maior = vet[0];
    posMaior = 0;
    for (int i=1;i<=n-1;i++){
        if (vet[i] > maior){
            maior = vet[i];
            posMaior = i;
        };
    };

    cout << fixed << setprecision(1);
    cout << "MAIOR VALOR = " << maior << endl;
    cout << "POSICAO DO MAIOR VALOR = " << posMaior << endl;

    return 0;
}

File: Linguagem C#/Vetores/MaiorPosicao.cs
using System;
using System.Globalization;

namespace MaiorPosicao {
    class Program {
        static void Main(string[] args) {

            CultureInfo CI = CultureInfo.InvariantCulture;

            int n,posMaior;
            double maiorValor;

            Console.Write("Quantos numeros voce vai digitar? ");
            n = int.Parse(Console.ReadLine());

            double[] vet = new double[n];

            for (int i = 0; i < n; i++) {
                Console.Write("Digite um numero: ");
                vet[i] = double.Parse(Console.ReadLine(), CI);
            };

            // Achando o maior valor e a sua posicao
            maiorValor = vet[0];
            posMaior = 0;
            for (int i = 1; i < n; i++) {
                if (vet[i] > maiorValor) {
                    maiorValor = vet[i];
                    posMaior = i;
                };
            };
            Console.WriteLine("");
            Console.WriteLine("MAIOR VALOR = "+maiorValor.ToString("F1",CI));
            Console.WriteLine("POSICAO DO MAIOR VALOR = "+posMaior);
        }
    }
}
